annotation-utils/src/main/java/edu/isi/vista/annotationutils/ExportAnnotations.kt

```diff
@@ -100,38 +125,27 @@ fun main(argv: Array<String>) {
                     .resultObjectThrowingExceptionOnFailure<AeroResult<AnnotatorRecord>>(mapper)
                     .body
 
-            val documentOutputDir = projectOutputDir.resolve(document.name)
```

Comment:
Why was this removed?

Reply:
The output structure had one directory per file so we flattened it into just one directory per project. Should we revert that?

Reply:
How do we handle multiple annotators annotating the same file on the same project?

Reply:
Oh is that possible? I thought each user has their own fresh copy of the project? (E.g. named `ArtifactExistence.DamageDestroy-bob_loblaw`)

Reply:
By policy, yes, but it is possible in the Inception data model for the someone else to come in an annotate on the project with someone else's name on it, so we should handle that case.

Reply:
The containing dir names the project owner, and the file itself names the annotator. Does that eliminate possibility of collision? For example there is `output/ArtifactExistence.DamageDestroy-gabbard/AFP_ENG_19960918.0012-admin.json`

Reply:
Yep, that will do the trick. Maybe add a comment somewhere to clarify this?